Userland/Libraries/LibGUI/Dialog.h

```diff
@@ -20,6 +20,19 @@ class Dialog : public Window {
         ExecYes = 3,
         ExecNo = 4,
     };
+    enum ScreenPosition {
+        CenterWithinParent = 0,
+
+        VerticalTop = 1,
+        VerticalMiddle = 2,
+        VerticalBottom = 4,
+
+        HorizontalLeft = 8,
+        HorizontalMiddle = 16,
+        HorizontalRight = 32,
+
+        Center = VerticalMiddle | HorizontalMiddle,
```

Comment:
if needed I can add prebaked values for other combinations.

Reply:
Personally I like sticking to the naming convention used in Gfx::TextAlignment, eg, TopLeft, CenterLeft, Center, CenterRight etc, but the maintainers can decide about that :) Great work!